src/js/cljs.js

```diff
@@ -117,7 +122,8 @@ export default function startClojureScriptEngine(opts: CLIOptsType): void {
 
   if (mainScript) {
     initClojureScriptEngine(opts);
-    return executeScript(mainScript, 'path');
+    executeScript(mainScript, 'path');
+    return socketRepl.close();
```

Comment:
This is just introducing an unnecessary dependency on `socketRepl`. 
We can take care of this in `cli.js`. If main script and only start the socket REPL if `opts.repl && opts['socket-repl']`, since a main script sets `repl` to false.